optimum/intel/openvino/utils.py

```diff
@@ -123,3 +126,21 @@ def _is_timm_ov_dir(model_dir):
         if hf_hub_id and model_info(hf_hub_id).library_name == "timm":
             return True
     return False
+
+
+def param_to_string(parameters) -> str:
+    """Convert a list / tuple of parameters returned from IE to a string."""
+    if isinstance(parameters, (list, tuple)):
+        return ', '.join([str(x) for x in parameters])
+    else:
+        return str(parameters)
+    
+
+def print_compile_model_properties(compile_model):
+    for property_key in compile_model.get_property('SUPPORTED_PROPERTIES'):
```

Comment:
Reference to https://github.com/openvinotoolkit/openvino/blob/master/tools/benchmark_tool/openvino/tools/benchmark/main.py#L447

Reply:
Done